docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -572,6 +576,10 @@ test('Title changed', async t => {
 Use this approach for Node.js callbacks that fire during the test run. To ensure that the test function
 does not finish before the callback is executed, suspend the test until the callback fires. For instance, you can introduce a Promise and wait until it completes synchronously, as shown in the example above.
 
+> The `boundTestRun` option requires the same test controller instance that is passed to the function used in a test declaration. It's unable to work with imported test controllers.
+>
+> The `boundTestRun` option is unable to pass a test controller instance to a function declared in another module or class.
```

Comment:
`boundTestRun` is so 'unable' :smile:
Maybe we can change at least one occurrence to 'does not accept' or something?

Reply:
And I'm not sure what this second note means

Reply:
I removed the redundancy